aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -1531,6 +1663,7 @@ REGISTER_DISPATCH(fake_quant_tensor_stub, &fake_quantize_tensor_kernel);
 REGISTER_DISPATCH(fake_quant_grad_tensor_stub, &fake_quantize_grad_tensor_kernel);
 REGISTER_DISPATCH(fake_quant_per_channel_stub, &fake_quant_per_channel_cpu);
 REGISTER_DISPATCH(fake_quant_grad_per_channel_stub, &fake_quant_grad_per_channel_cpu);
+REGISTER_DISPATCH(LayerNormKernelQuantized, &LayerNormKernelQuantizedImpl);
```

Comment:
nit: adjective before noun reads better IMO, so `QuantizedLayerNormKernel`. Also it would be better to match the style of the surrounding code and name it in snake_case